### DISCUSSION TITLE
Is there any documentation on running Benthos inside Google Cloud Platform?

### DISCUSSION THREAD

**Discussion by jarrodhroberson** [redpanda-data/connect#1852] [Q&A]
preferably Google App Engine, but Cloud Run would work as well for testing to see if it is appropriate for my use case.

**Comment by mihaitodor** [comment]
Hey @jarrodhroberson, according to [the docs](https://cloud.google.com/run/docs/deploying), you can use Cloud Run to deploy any Docker image. Benthos does provide a Docker image under [`jeffail/benthos`](https://hub.docker.com/r/jeffail/benthos). Give it a shot and let me know if that doesn't work.
